comps/comps-foreman-plugins-rhel6.xml

```diff
@@ -78,6 +77,8 @@
       <packagereq type="default">tfm-rubygem-safe_yaml</packagereq>
       <packagereq type="default">tfm-rubygem-wicked</packagereq>
       <packagereq type="default">tfm-rubygem-zscheduler</packagereq>
+      <packagereq type="default">rubygem-concurrent-ruby</packagereq>
```

Comment:
No need to add this to EL6, it isn't used by anything (it's only in EL7/F21 as the smart_proxy_dynflow plugin uses it.)